lib/tasks/find_possible_duplicate_accounts.rake

```diff
@@ -0,0 +1,64 @@
+namespace :accounts do
+  namespace :duplicate_accounts do
+    desc "Find possible duplicate acccounts"
+    task :find => [:environment] do
+      CSV.open("duplicate_users_by_name.csv", 'w+') do |csv|
+        csv << ['User First Name', 'User Last Name', 'Email Address(es)', 'User ID', 'Social Signup Successful', 'Reset Password Help Requested', 'Help Request Failed']
+
+        where_names_match = User.joins{ User.unscoped.as(same)
+                        .on{
+                          (same.id != ~id) & ((lower(same.first_name) == lower(~first_name)) & (lower(same.last_name) == lower(~last_name)))
+                        }
+                      }.uniq.order(id: :asc)
+
+        where_names_match.find_each do |user|
+          csv << ["#{user.first_name}",
+                  "#{user.last_name}",
+                  "#{user.contact_infos.map(&:value).join(", ") if user.contact_infos.any?}",
```

Comment:
Nope. Columns with commas in CSV files are just put into quotes.
`Robert,Martin,"c2428a@48b89c.e9f831, f28661@a4a961.b062a5",3,true on: 2016-09-13 17:34:43 UTC,true on: 2016-09-13 17:34:43 UTC. true on: 2016-09-13 17:34:43 UTC,true on: 2016-09-13 17:34:43 UTC`